deploy/helm/magda/charts/minio/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
You might have told me this before the holidays and I've forgotten, but what's the advantage of copy-pasting the chart in vs. just referring to its URL as a dependency?

Reply:
Minio here is not just a dependency, it's also a magda service in a way (since storage needs a minio server running for it to work). Is it possible to achieve that just by referring to the chart via a URL?

Reply:
It _should_ be but we haven't tried it in Magda yet - ideally Helm should work a bit like NPM  but for K8S - you refer to a chart which can refer to other charts that it needs, and so on.